cypress/e2e/hide-question-numbers-interactives.ts

```diff
@@ -0,0 +1,149 @@
+import ActivityPage from "../support/elements/activity-page";
+
+const activityPage = new ActivityPage;
+
+context("Activity hide question numbers checked", () => {
+  before(() => {
+    cy.visit("?activity=sample-activity-hide-question-number");
+    activityPage.getActivityTitle().should("contain", "Test Hide Question Number Setting");
+  });
+  describe("Hide question numbers checked in activity",() => {
+    it.only("Verify the left column has the hide question number setting OFF and the right has it ON.",()=>{
+      activityPage.clickPageItem(0);
+      cy.wait(2000);
```

Comment:
Yes! These waits are all over the repository, so I was wondering about that. I'll try it again without the waits. Saw this and thought maybe alternative statements might cover it (to get to page 1, 2, 3, 4 in the activity, etc.):
https://www.lambdatest.com/blog/cypress-best-practices/

Reply:
Update: I tried removing the waits in this test, but that completely broke it. :( Kirk might have some ideas for removing unnecessary waits in the repository, so it could be worth exploring, though that might need to be addressed separately for now.

Reply:
In this particular case there should be a statement that asserts page 1 is being shown. Without that `activityPage.getInteractive().eq(0).find(".has-question-number .header div").should("exist")` will just match the first interactive on any page.